src/GatewayService.php

```diff
@@ -34,6 +34,8 @@
 ////////////////////////////////////////////////////////////////////////////////
 //
 
+use phpDocumentor\Reflection\Types\Boolean;
```

Comment:
this import isnt used, it could be removed

Reply:
@thomaspicquet, thank you for the observation. 
We are going to shortly merge a Pull Request which is removing this dependency.